framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceImplementation.java

```diff
@@ -256,6 +256,10 @@ private void updateInferredExecutableParameterTypes(
       AnnotatedTypeMirror receiverArgATM = atypeFactory.getReceiverType(invocationTree);
       AnnotatedExecutableType methodDeclType = atypeFactory.getAnnotatedType(methodElt);
       AnnotatedTypeMirror receiverParamATM = methodDeclType.getReceiverType();
+      if (receiverParamATM == null) {
+        // receiver type is null for static methods and constructors
+        return;
```

Comment:
I don't think returning here is the right choice, because that will cause inference to skip inference for the rest of the parameters, too. I think this should just skip the rest of this block, instead.

Reply:
I replaced return statement with the rest of this block.